xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -373,7 +390,11 @@ void CRenderer::CreateSubtitlesStyle()
   else
     m_overlayStyle->assOverrideStyles = KODI::SUBTITLES::OverrideStyles::DISABLED;
 
-  m_overlayStyle->marginVertical = GetSubtitleVerticalMargin();
+  // Changing vertical margin while in playback causes side effects when you
```

Comment:
Not against this but probably requires deeper investigation? Probably some issue in libass itself?

Reply:
i tried check libass code but i found nothing i could understand
i can try to report it but we already know how it will end...